test/PowerShellEditorServices.Test.Host/ServerTestsBase.cs

```diff
@@ -11,6 +11,7 @@
 using System.Diagnostics;
 using System.IO;
 using System.Text;
```

Comment:
Can you add this right here:
```
#if CoreCLR
using System.Reflection;
#endif
```
Without this, my build within VS 2017 fails with:
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS1061	'Type' does not contain a definition for 'GetTypeInfo' and no extension method 'GetTypeInfo' accepting a first argument of type 'Type' could be found (are you missing a using directive or an assembly reference?)	PowerShellEditorServices.Test.Host(netcoreapp2.0)	C:\Users\Keith\GitHub\rkeithhill\PowerShellEditorServices\test\PowerShellEditorServices.Test.Host\ServerTestsBase.cs	42	Active
```